guide/C/ch_txns.xml

```diff
@@ -1029,7 +1029,7 @@
       create it for you.
     </para>
 
-    <para>In this howto, we&rsquo;ll take a monthly Internet subscription of 20 USD as example, which is taken
+    <para>In this how-to, we&rsquo;ll take a monthly Internet subscription of 20 USD as example, which is taken
```

Comment:
IMHO we should rename all `howto`s to exercise to match with `tutorial` in the doc name.

Reply:
I can't understand the difference between them, because I'm not a native English user.

Reply:
Me too. That's why I suggest different term.
@jralls our native speaker

Reply:
I like exercise.

The usage manuals will tell you to use how-to, but they're old and I believe it's pretty common to leave out the hyphen and Apple's spell checker doesn't complain about it.

Reply:
I can understand what you say.
However, it is important to unify the terms in order to improve the quality of software and documents.

You may change to exercise soon, but for now it's better to unify with how-to, I think.